data/private

```diff
@@ -150,10 +152,14 @@ tendawifi.com
 
 # TP-Link router
 tplinkwifi.net
+tplogin.cn
```

Comment:
Use the `full:` prefix whenever you can to avoid mismatch.

Reply:
Same as above.

Reply:
This domain has no A/AAAA/CNAME records via `nslookup` and can't get echo from `ping`, only used for intranet, so I leave it untouched.

Reply:
> This domain has no A/AAAA/CNAME records via `nslookup` and can't get echo from `ping`, only used for intranet, so I leave it untouched.

We should avoid mismatches whenever we can. What if someone, who owns this domain, decides to make use of a sub-domain under it in the future?
Making changes based on solid proofs would be better.

Reply:
https://resource.tp-link.com.cn/m/productClass/product_document?id=1655112502304621

Added `full:tplogin.cn`.